mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb

```diff
@@ -482,6 +482,10 @@ func resourceContainerNodePoolDelete(d *schema.ResourceData, meta interface{}) e
 	//Check cluster is in running state
 	_, err = containerClusterAwaitRestingState(config, nodePoolInfo.project, nodePoolInfo.location, nodePoolInfo.cluster, userAgent, d.Timeout(schema.TimeoutCreate))
 	if err != nil {
+		if isGoogleApiErrorWithCode(err, 404) {
+			log.Printf("[INFO] GKE node pool %s doesn't exist to delete", d.Id())
```

Comment:
I noticed that I made a mistake when sending this PR. 

cl/371172872 was correct, but when I sent this OSS PR, the change was already [made in OSS](https://github.com/GoogleCloudPlatform/magic-modules/blob/a8f275027161814ea209758c0247bdffd91dd708/mmv1/third_party/terraform/resources/resource_container_node_pool.go.erb#L494-L498) (just a few lines below). Since the code change in this PR wasn't what I intended to make, we may just revert this PR.

On the other hand, different from cl/371172872. This OSS code had an extra `containerClusterAwaitRestingState` before `containerNodePoolAwaitRestingState`. I wonder if we would introduce a similar bug if we simply revert this PR-- when deleting a node pool, should we raise an error if the cluster doesn't exist, or should we just print a log? If it's the latter, then we may fix this log message. 

@rileykarson @ScottSuarez WDYT?

Reply:
If the cluster doesn't exist we can assume the node pool doesn't. It's not immediately obvious what's wrong here to me.

Reply:
sounds good. Maybe I shouldn't say "fix this log message". Since we assume the node pool doesn't exist because the cluster doesn't, we can improve this log message to `"[INFO] GKE cluster %s doesn't exist, skipping node pool %s deletion", nodePoolInfo.cluster, d.Id()`.

It's not that critical, just the code wasn't doing what I thought it would do, and I think maybe it would be confusing to others as well.

Reply:
Good point, we could make it more clear! If you're interested in changing it feel free, but I don't feel it's required.

Reply:
will do. Thanks